lib/cli/driver.js

```diff
@@ -262,9 +274,88 @@ class DriverCommand {
     return this.config.installedDrivers;
   }
 
-  update (/*{driver}*/) {
-    // TODO implementation
-    return {};
+  async update ({driver, unsafe}) {
+    const all = driver === UPDATE_ALL;
+    // if we're specifically requesting an update for a driver, make sure it's installed
+    if (!all && !this.config.isInstalled(driver)) {
+      throw new Error(`Driver '${driver}' was not installed, so can't be updated`);
+    }
+    const driversToUpdate = all ? Object.keys(this.config.installedDrivers) : [driver];
+    const errors = {};
```

Comment:
it makes sense to either add a comment about what will be keys ands values of these maps